catalyst_voices/packages/catalyst_voices_models/lib/src/seed_phrase.dart

```diff
@@ -6,10 +6,36 @@ import 'dart:typed_data';
 import 'package:bip39/bip39.dart' as bip39;
 import 'package:bip39/src/wordlists/english.dart';
 import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+import 'package:collection/collection.dart';
 import 'package:convert/convert.dart';
 import 'package:ed25519_hd_key/ed25519_hd_key.dart';
 import 'package:equatable/equatable.dart';
 
+/// Represent singular mnemonic words which keeps data as well as index of
+/// this word.
+///
+/// This is useful because seed phrase may have duplicate words. With this
+/// structure we can distinguish which word is which.
+final class SeedPhraseWord extends Equatable
+    implements Comparable<SeedPhraseWord> {
+  /// Word value.
+  final String data;
+
+  /// Number of this word in [SeedPhrase].
+  final int nr;
```

Comment:
Call it index? Programmers use indexes, kinda seems more familiar than a number, what do you think?

Reply:
its intentionally `nr` and not `index`. 
My reasoning was that we do stuff like `shuffling`, after that it won't make sense to have `SeedPhraseWord` with `index:0` in list at index `9` but having `SeedPhraseWord` nr. `1` at index `9` makes little more sense.